frontend/src/components/explorer/mapViewer/Svgmap.vue

```diff
@@ -162,28 +159,33 @@ export default {
       await this.$store.dispatch('maps/getSvgMap', payload);
       this.bindKeyboardShortcuts();
       this.applyLevelsOnMap();
+      this.setupHoverEventHandlers();
     },
     setupHoverEventHandlers() {
-      const self = this;
+      // construct list of classes to be selected with jquery $(".class1,.class2,.class3,...")
```

Comment:
I am bit curious, why we are using jQuery? Shouldn't plain JS suffice? Below in the same file we are using `document.querySelector`

Reply:
Not sure tbh, we modified existing code when we made these changes. Feel free to create an issue for replacing jQuery, I don't think we should make more changes in this PR if possible

Reply:
There is already [an issue to remove JQuery](https://github.com/MetabolicAtlas/private-issues/issues/28), so if would be great if the use of this library can be kept to a minimum. It's not a must to resolve this in the current PR, just a thing to keep in mind in the future.